### PR TITLE
Feature: reconcile search with designs [NP-895]

### PR DESCRIPTION
**Pull request by davidsauntson**
Reason for change:
Designs for search had changed since they were first implemented in the design system.  This PR updates the style of the search bar to match those in the design file.

Changes:
 - remove icon in mobile search button and replace with text
 - added VRT for expanded header
 - update VRT references

### CODE REVIEW

**Review by baseonmars — DISMISSED** [review]

File: scss/6-components/_navigation.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
 @import 'navigation_greedy-nav';
 
-nav,
```
Comment (davidrapson) [inline scss/6-components/_navigation.scss:3]: 💯

**Review by davidrapson — DISMISSED** [review]
👍

**Review by soniaturcotte — APPROVED** [review]
Looks good for search page! 👍  thanks for doing that. Just to confirm this won't affect the header?

**Comment by davidsauntson** [comment]
> Looks good for search page! 👍 thanks for doing that. Just to confirm this won't affect the header?

@soniaturcotte Collapsed it is the same, expanded it will show 'Search' instead of the icon:
image

**Comment by davidsauntson** [comment]
> > Looks good for search page! 👍 thanks for doing that. Just to confirm this won't affect the header?
> 
> @soniaturcotte It will show Search instead of the icon in the header on desktop as it currently does.
> 
> On mobile, collapsed it is the same, expanded it will show 'Search' instead of the icon.
> image

I've changed this now so it's back to the search icon on mobile, and added a VRT to cover this.

**Review by davidrapson — APPROVED** [review]
👍 Looks great. 😍 Thank you for removing the default element styles

**Pull request merged**